noir-projects/aztec-nr/aztec/src/messages/discovery/partial_notes.nr

```diff
@@ -72,9 +67,9 @@ pub unconstrained fn process_partial_note_private_msg(
         .push(pending);
 }
 
-/// Searches for public logs that would result in the completion of pending partial notes, ultimately resulting in the
-/// notes being delivered to PXE if completed.
-pub unconstrained fn fetch_and_process_public_partial_note_completion_logs<Env>(
+/// Searches for logs that would result in the completion of pending partial notes, ultimately resulting in the notes
+/// being delivered to PXE if completed.
+pub unconstrained fn fetch_and_process_partial_note_completion_logs<Env>(
```

Comment:
I think a nice addition to this function's docs would be an explanation of why do we use capsule arrays here in the first place (to get the behavior of unbonded dynamic array which Noir doesn't support). This pattern might feel very twisted for readers not intimate with capsules and capsule arrays.